File: product.cpp
#include "product.h"

product::product(const std::string &name, double price, int item_availability)
    : name(name), price(price), item_availability(item_availability) {}

std::string product::get_name() const { return name; }
double product::get_price() const { return price; }
int product::get_item_availability() const { return item_availability; }
void product::update_item_availability(int quantity) { item_availability += quantity; }

File: product.py
class Product:
    def __init__(self, name, price, item_availability):
        self._name = name
        self._price = price
        self._item_availability = item_availability

    def get_item_name(self):
        return self._name

    def get_price(self):
        return self._price

    def get_item_availability(self):
        return self._item_availability

    def update_item_availability(self, quantity):
        self._item_availability += quantity
